src/errorpage.cc

```diff
@@ -428,7 +440,7 @@ TemplateFile::loadFromFile(const char *path)
 
     if (fd < 0) {
         /* with dynamic locale negotiation we may see some failures before a success. */
-        if (!silent && templateCode < TCP_RESET) {
+        if (!silent && templateCode < TCP_RESET && !FindDefaultTemplate(templateCode)) {
```

Comment:
However, another (level 3) error message is always printed when open() fails :
```
2022/10/29 23:05:36.988| 50,3| fs_io.cc(60) file_open: error opening file /home/ed/work/squid-pool/1/share/errors/templates/MGR_INDEX: (2) No such file or directory
```

Reply:
Let's ignore that small level-3 problem for now.

Reply:
Ok.